web_transport/sdk/impl/web_transport_server_backend.cc

```diff
@@ -32,9 +32,15 @@ void WebTransportServerBackend::OnSessionReady(
   std::unique_ptr<WebTransportServerSession> wt_session =
       std::make_unique<WebTransportServerSession>(session, http3_session,
                                                   io_runner_, event_runner_);
-  sessions_[session->id()] = std::move(wt_session);
+  WebTransportServerSession* session_ptr = wt_session.get();
+  if (sessions_.count(http3_session->connection_id().ToString()) > 0) {
+    LOG(WARNING) << "Session with the same connection ID exits, the old one "
+                    "will be terminated. Only one WebTransport session for a "
+                    "QUIC connection is supported.";
+  }
+  sessions_[http3_session->connection_id().ToString()] = std::move(wt_session);
```

Comment:
Where do you remove the session id from the map?

Reply:
It's not being removed. I'll file an issue and fix it later.